images/prometheus-pushgateway-bitnami/tests/main.tf

```diff
@@ -0,0 +1,51 @@
+terraform {
+  required_providers {
+    oci  = { source = "chainguard-dev/oci" }
+    helm = { source = "hashicorp/helm" }
+  }
+}
+
+variable "digests" {
+  description = "The image digests to run tests over."
+  type = object({
+    prometheus-pushgateway = string
+  })
+}
+
+data "oci_string" "ref" {
+  for_each = var.digests
+  input    = each.value
+}
+
+data "oci_exec_test" "healthy" {
+  digest = var.digests["prometheus-pushgateway"]
+  script = "${path.module}/02-healthy.sh"
+}
+
+resource "random_pet" "suffix" {}
+
+resource "helm_release" "prometheus_community_pushgateway" {
+  name             = "pushgateway-${random_pet.suffix.id}"
+  repository       = "https://prometheus-community.github.io/helm-charts"
```

Comment:
Can this use a bitnami helm chart?

Reply:
i did not find a bitnami helm chart for that